areas.wotw

```diff
@@ -7359,6 +7389,9 @@ anchor UpperDepths.RightKeystonePath at 305, -4426:  # at the ledge below the en
       DepthsLight, DoubleJump OR Dash OR Glide OR Sword  # use the poles hanging from the ceiling
       DepthsLight, Bash, Grenade=1
       DepthsLight, SentryJump=1
+    kii:
+      Bash
+      DepthsLight, Hammer, OR Shuriken=2 OR Sentry=2 OR Sword
```

Comment:
Not sure if this is supposed to be
`DepthsLight, Hammer, Shuriken=2 OR Sentry=2 OR Sword` or
`DepthsLight, Hammer OR Shuriken=2 OR Sentry=2 OR Sword`

Reply:
DepthsLight, Hammer OR Shuriken=2 OR Sentry=2 OR Sword